modules/ws_bridge/BUILD

```diff
@@ -0,0 +1,65 @@
+# =================================================================================================
+# Copyright (C) 2023-2024 HEPHAESTUS Contributors
+# =================================================================================================
+
+load("@hephaestus//bazel:hephaestus.bzl", "heph_cc_binary", "heph_cc_library", "heph_cc_test")
+
+package(default_visibility = ["//visibility:public"])
+
+heph_cc_library(
+    name = "ws_bridge",
```

Comment:
I would probably call this `foxglove_bridge` as we use foxglove specific library, otherwise I would spell out to `websocket_bridge`

Reply:
ok with websocket_bridge. Especially since we are using it also without foxglove (and it is useful that way), I am leaning a bit more toward not calling it foxglove_bridge. wdyt?

would you also go for the namespace heph::websocket_bridge?
way too long IMO and since the module name was used as namespace in create_module.py, I thought I keep the module name also a bit more concise.

Reply:
in general I prefer long descriptive names as it massively reduce the cognitive load to understand what's going on.
Regarding the namespace, I think in this case it is not a problem to have a long one, as this is going to be an app, thus all the code that uses it already inside the namespace. 
If it were a library (to be used in other parts of heph) I would have agreed for a shorter namespace